### PR TITLE
Updated OSSEC readme and defaults

### PR DESCRIPTION
**Pull request by dolanjs**
OSSEC updated their download repository and now they do support HTTPS on the download page. They still do not provide a signed download just the MD5 and SHA1 checksums. The download url and download names also changed now that OSSEC is hosted on github.
- Updated default values needed for ansible playbook for OSSEC version 2.8.2
- Updated task to unpack the download to reflect the new upstream naming convention.
- Updated README with notes about determining the `download_checksum`

### CODE REVIEW

File: ansible/roles/build-ossec-deb-pkg/tasks/main.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@
   with_items: build_ossec_deb_pkg_dependencies
 
 - name: extract OSSEC archive to /tmp
-  shell: tar -xvf {{ build_path }}/{{ download_name }} -C /tmp
+  shell: tar -xvf {{ build_path }}/ossec-hids-{{ download_name }} -C /tmp
```
Comment (garrettr) [inline ansible/roles/build-ossec-deb-pkg/tasks/main.yml]: Do you also need to update the other locations where `{{ download_name }}` is used in the task? Why did you move `ossec-hids` from `download_name` to inside the shell call?

Comment (dolanjs) [inline ansible/roles/build-ossec-deb-pkg/tasks/main.yml]: The new download url is: `https://github.com/ossec/ossec-hids/archive/2.8.2.tar.gz` The downloaded file is `ossec-hids-2.8.2.tar.gz`

Think this will be getting re-org'd by @conorsch at some point in the future.

Comment (dolanjs) [inline ansible/roles/build-ossec-deb-pkg/tasks/main.yml]: I moved `ossec-hids` out of the `download_name` because the `download_url` + `download_name` otherwise would not match the new url.

**Comment by conorsch** [comment]
1. We should link to the [OSSEC downloads page](http://www.ossec.net/?page_id=19) in the README. Now that there's a wazuh apt repo, we should be explicit about where these packages are being pulled from. Providing a direct link also encourages folks to verify the checksums.
2. The checksum vars should be named consistently and explicitly:
   -  `download_checksum` -> `download_checksum_sha256`
   -  `download_md5` -> `download_checksum_md5`

Listing the current vars in the README is a great idea. :+1:

**Comment by conorsch** [comment]
lgtm!

**Comment by garrettr** [comment]
@dolanjs lgtm! I'm really happy we got all of this documented. Can you squash these 4 commits into 1? Then I'll merge it.
